specs/IsNonClientRegionSupportEnabled.md

```diff
@@ -0,0 +1,141 @@
+webView.CoreWebView2.NavigationStarting += SetNonClientRegionSupport;
+
+private void SetNonClientRegionSupport(CoreWebView2 sender, CoreWebView2NavigationStartingEventArgs args)
+{
+    var coreWebView2Settings = webView.CoreWebView2.Settings;
+    var urlCompareExample = "www.microsoft.com";
+    var uri = new Uri(args.Uri);
+
+    if (String.Equals(uri.Host, urlCompareExample, StringComparison.OrdinalIgnoreCase) &&
+        !coreWebView2Settings.IsNonClientRegionSupportEnabled)
+    {
+        coreWebView2Settings.IsNonClientRegionSupportEnabled = true;
+    }
+    else if (!String.Equals(uri.Host, urlCompareExample, StringComparison.OrdinalIgnoreCase) && 
+        coreWebView2Settings.IsNonClientRegionSupportEnabled)
+    {
+        coreWebView2Settings.IsNonClientRegionSupportEnabled = false;
+    }
+}
+```
+
+# Remarks
+If the feature flag (`msWebView2EnableDraggableRegions`) is used to enable the CSS style `app-region`'s
+values `drag` and `no-drag`, draggable region support will remain enabled 
+even if the `IsNonClientRegionSupportEnabled` setting is `FALSE`. 
+* Note: The feature flag is experimental and should not be used in production.
+
+# API Notes
+See [API Details](#api-details) section below for API reference.
+
+## Declaring Non-client App Regions
+Non-client regions are HTML elements that are marked with the css style `app-region`.
+* Draggable regions can be declared through the values `drag` or `no-drag`.  
+
+# API Details
+```cpp
+/// This is the ICoreWebView2Settings Staging interface.
+[uuid(436CA5E2-2D50-43C7-9735-E760F299439E), object, pointer_default(unique)]
+interface ICoreWebView2Settings12 : ICoreWebView2Settings11 {
+  /// `IsNonClientRegionSupportEnabled` property enables web pages to use the 
+  /// `app-region` CSS style. Disabling/Enabling the `IsNonClientRegionSupportEnabled`
+  /// takes effect after the next navigation. Defaults to `FALSE`.
+  /// 
+  /// When this property is `TRUE`, then all the following non-client region support 
+  /// will be enabled:
+  /// 1. Draggable Regions will be enabled and treated as a window's title bar. 
+  /// Draggable Regions are non-client regions on a webpage that are exposed through the css
+  /// attribute `app-region` and can take the values `drag` or `no-drag`. When set to 
+  /// `drag`, these regions will be treated like the window's title bar, supporting 
+  /// dragging of the entire WebView and its host app window, the title bar context menu
+  /// upon right click, and the maximizing to fill the window and restoring the window size
+  /// upon double click of the html element. 
+  ///
+  /// When set to `FALSE`, then all non-client region support will be disabled. Web
+  /// pages will not be able to use the `app-region` CSS style.
+  [propget] HRESULT IsNonClientRegionSupportEnabled([out, retval] BOOL* enabled);
+  /// Set the IsNonClientRegionSupportEnabled property
+  [propput] HRESULT IsNonClientRegionSupportEnabled([in] BOOL enabled);
+}
+```
```

Comment:
Please also include the MIDL3 for your changes. You can find it in the generated Microsoft.Web.WebView2.Core.IDL file generated after running build-apiwriter.ps1.